compose/backend.yml

```diff
@@ -79,18 +79,43 @@ services:
       ELASTIC_SEARCH_HOSTS: ccd-elasticsearch:9200
       ELASTIC_SEARCH_DATA_NODES_HOSTS: "http://ccd-elasticsearch:9200"
       # Uncomment this line to enable JVM debugging and uncomment the port mapping below
-      # JAVA_TOOL_OPTIONS: '-XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0 -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005'
+      #JAVA_TOOL_OPTIONS: '-XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0 -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005'
     ports:
       - 4452:4452
       # Uncomment this and the JAVA_TOOL_OPTIONS flag for JVM debugging
-      # - 5005:5005
+      #- 5005:5005
     depends_on:
       - ccd-shared-database
       - ccd-user-profile-api
       - ccd-definition-store-api
     networks:
       - ccd-network
 
+  ccd-case-document-am-api:
```

Comment:
`backend.yml` is reserved for core CCD services that are started by default with `./ccd compose up`
ccd-case-document-am-api should not be here. It would be better to place is in:
`dm-store.yml`
So that it's started only when dm store is started

Reply:
I suggest we add to README too

Reply:
Yes, I have moved case document setup in dm-store.yaml fie.

Reply:
Yes, Nathan we are trying to consolidate all the changes once our testing is done read me file will be updated accordingly.

Reply:
@dharmendra8186 , @nitishsingla1234 , @aashishaggarwal123  - Could you please confirm the above comments are actioned fully and README updated precisely as per the latest facts?

Reply:
yes, all the comments are incorporated and updated read me file is available in the PR.